language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/CommandClient.java

```diff
@@ -17,7 +18,15 @@
 
 /** An RxJava version of {@link com.daml.ledger.api.v1.CommandServiceGrpc} */
 public interface CommandClient {
-
+  Single<Empty> submitAndWait(CommandsSubmission params);
```

Comment:
Am I missing something or are all the other methods mentioned in the deprecation notices (e.g. `submitAndWaitForResult`) still missing?

Reply:
They are there, further down 😃